src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -155,7 +155,7 @@ private void addBinaryToPath(String hdfsBinaryPath) throws IOException {
     BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
     bufferedWriter.write(scriptContent);
     bufferedWriter.close();
-    Runtime.getRuntime().exec("chmod a+x " + pathEnvVarLocation);
+    new ProcessBuilder("chmod", "a+x", pathEnvVarLocation).start();
```

Comment:
Instead of method chaining, it is better java style to create a separate process builder object then call start on a separate line.